bundles/org.openhab.binding.keba/README.md

```diff
@@ -48,6 +48,33 @@ All devices support the following channels:
 | maxpilotcurrent         | Number:ElectricCurrent   | yes       | current offered to the vehicle via control pilot signalization          |
 | maxpilotcurrentdutycyle | Number:Dimensionless     | yes       | duty cycle of the control pilot signal                                  |
 
+## Rule Actions
+Certain Keba models support setting the text on the built in display.
+The text cannot be set via a rule action `setDisplay`. It comes in two variants:
+
+```java
+
+rule "Set Display Text"
+when
+  System reached start level 100
+then
+   val keContactActions = getActions("keba", "keba:kecontact:1")
+   // Default duration
+   keContactActions.setDisplay("TEXT$1")
+   // Explicit duration set
+   keContactActions.setDisplay("TEXT$2", 5, 10)
+end
+
+```
+
+
+| Parameter  | Description                                                                                                                                |
```

Comment:
Could markdowntable.com or your built-in IDE markdown tool be used to align the table?

Reply:
Sure, done.